flag_engine/identities/models.py

```diff
@@ -28,6 +28,10 @@ class IdentityModel:
     identity_features: typing.List[FeatureStateModel] = field(default_factory=list)
     identity_traits: typing.List[TraitModel] = field(default_factory=list)
 
+    @property
+    def pk(self):
```

Comment:
I wonder if this should be named something other than `pk` since it means something in Djang. Unless there's a reason it needs to be `pk`?

Reply:
call it dynamo_pk?

Reply:
I think maybe something more meaningful / explicit like `composite_key` or something?

Reply:
Hmm, but it's literally the primary key for the dynamo document. but I am not sure if engine needs to be aware of dynamo and if this is a leaky abstraction?

Reply:
I like `composite_key`!